serve/server_test.go

```diff
@@ -33,20 +33,20 @@ func TestGreeterSample(t *testing.T) {
 	out := &bytes.Buffer{}
 
 	unaryWant := `
-Header: map[content-type:[application/grpc]]
+Header: map[content-type:[application/grpc] trailer:[Grpc-Status Grpc-Message Grpc-Status-Details-Bin]]
```

Comment:
the http version of the grpc server predeclares known trailers https://github.com/grpc/grpc-go/blob/ebc30b8fc32e02f0ac5eff0ddbe4ccdca58bd8c4/internal/transport/handler_server.go#L257-L264

Reply:
I think they do. They are the contents of the `Status` proto when you return an error. The grpc lib might be stripping them out.